### ISSUE TITLE
Cannot find module '@angular-devkit/schematics/tools' on a fresh install

### ISSUE THREAD

**Issue by sylvainar** [jscutlery/semver#237]
I'm using Nx to manage node packages (no dependency to angular), but I can't setup this plugin on a fresh install, see error below:
```
✔ Lock/sync versions between packages? (y/N) · true
✔ Enforce conventional commits? (Y/n) · false
Cannot find module '@angular-devkit/schematics/tools'
Require stack:
- /home/sylvain/Workspace/app-common/node_modules/@nrwl/tao/src/commands/ngcli-adapter.js
- /home/sylvain/Workspace/app-common/node_modules/@nrwl/tao/src/commands/generate.js
- /home/sylvain/Workspace/app-common/node_modules/@nrwl/tao/index.js
- /home/sylvain/Workspace/app-common/node_modules/@nrwl/cli/lib/init-local.js
- /home/sylvain/Workspace/app-common/node_modules/@nrwl/cli/bin/nx.js
```

My dev-dependencies:
```
"devDependencies": {
    "@jscutlery/semver": "^2.4.0",
    "@nrwl/cli": "12.5.7",
    "@nrwl/eslint-plugin-nx": "12.5.7",
    "@nrwl/jest": "12.5.7",
    "@nrwl/linter": "12.5.7",
    "@nrwl/node": "^12.5.7",
    "@nrwl/tao": "12.5.7",
    "@nrwl/workspace": "12.5.7",
    "@types/jest": "^26.0.24",
    "@types/node": "14.14.33",
    "@typescript-eslint/eslint-plugin": "4.19.0",
    "@typescript-eslint/parser": "4.19.0",
    "dotenv": "8.2.0",
    "dotenv-cli": "^4.0.0",
    "eslint": "7.22.0",
    "eslint-config-airbnb-typescript": "^12.3.1",
    "eslint-plugin-import": "^2.22.0",
    "jest": "27.0.3",
    "ts-jest": "27.0.3",
    "ts-node": "~9.1.1",
    "typescript": "~4.2.4"
  }
  ```

**Comment by daton89** [comment]
I also have this error, I'm using react

**Comment by edbzn** [comment]
This will be fixed soon, a PR is open #238, we are just waiting for the next Nx version which comes with a fix that we need. Meanwhile, a quick fix would be to install the dep `@angular-devkit/schematics` before the install, then you can remove it.

**Comment by daton89** [comment]
cool! thank for the quick support!
